_licenses/cddl-1.0.txt

```diff
@@ -0,0 +1,378 @@
+---
+title: Common Development and Distribution License 1.0
+spdx-id: CDDL-1.0
+
+description:
+
+how: Create a text file (typically named LICENSE or LICENSE.txt) in the root of your source code and copy the text of the license into the file.
+
+using:
+  - OpenZFS: http://open-zfs.org
+  - NetBeans: https://netbeans.org/
+  - GlassFish: https://github.com/eclipse-ee4j/glassfish
```

Comment:
The values should refer to license files that could be detected, see longer explanation at https://github.com/github/choosealicense.com/pull/656#pullrequestreview-228073804

Reply:
This is a tricky one. Netbeans for example historically was CDDL, but it has moved to the Apache project and has been relicensed as Apache 2. Glassfish move away from a CDDL license it turns out to the Eclipse license, see https://github.com/apache/incubator-netbeans. 

I was able to find some archived projects like https://github.com/javaee/javax.annotation, which are also moving to the Eclipse project and seem to be relicensed as part of that as well. 

What's the best approach here? Use some archived repositories like https://github.com/javaee/javax.annotation? Existing software can often depend on the previous versions of these projects and the move is still in progress, so I'm not sure which ones to use here then.

Reply:
https://github.com/javaee/javax.annotation/blob/master/LICENSE specifically won't work since it is CDDL and GPL concatenated. Archived examples are fine if there aren't others. If there aren't examples that could be detected, there's really no reason to catalog the license here. Certainly nobody should be choosing it for new projects.